looknfeel/src/main/less/skin/skin.config.less

```diff
@@ -351,7 +351,6 @@
 @tablesPanel-table-line-shadowColor: #c2c2c2;//rgba(0, 0, 0, 11%);
 @tablesPanel-table-line-margin : 0 0 @h*2 0;
 @tablesPanel-table-currentLine-arrow-width: 20px;
-@tablesPanel-table-currentLine-arrow-bg: "arrow-current.png";
```

Comment:
arrow-current.png used somewhere else ? If not, I guess we can also delete the file ?